yt_dlp/extractor/videocampus_sachsen.py

```diff
@@ -0,0 +1,98 @@
+# coding: utf-8
+from .common import InfoExtractor
+
+
+class VideocampusSachsenIE(InfoExtractor):
+    _VALID_URL = r'https?://videocampus\.sachsen\.de/(?:(?:m/(?P<tmp_id>[0-9a-f]+))|(?:(?P<c>category/)?video/(?P<display_id>[A-Za-z0-9-]+)/(?P<id>[0-9a-f]{32}))(?(c)/[0-9]+))'
```

Comment:
```suggestion
    _VALID_URL = r'''(?x)https?://videocampus\.sachsen\.de/(?:
        m/(?P<tmp_id>[0-9a-f]+)|
        (?:category/)?video/[\w-]+/(?P<id>[0-9a-f]{32})
    )'''
```
(untested)

Reply:
So you'd prefer dropping the extraction of `display_id`?

Not sure about `[\w-]` since URLs are ASCII only normally; however I don't have a strong preference.

> (untested)

Works fine.

Reply:
> So you'd prefer dropping the extraction of display_id?

Dont capture fields that are not used